docs/HACKING_QUICKSTART.md

```diff
@@ -108,20 +108,28 @@ This is how my projects are laid out:
 
 ```
 ~/my-projects/servo/
-~/my-projects/cocoa-rs/
-~/my-projects/glutin/
+~/my-projects/webrender/webrender
 ```
 
-These are all git repositories.
+To make it so that servo uses `~/my-projects/webrender/webrender`, first ascertain which version of the crate Servo is using and whether it is a git dependency or one from crates.io.  
 
-To make it so that servo uses `~/my-projects/cocoa-rs/` and `~/my-projects/glutin/` , create a `~/my-projects/servo/.cargo/config` file:
+Both information can be found using, in this exampple, `/mach cargo pkgid webrender`.
 
-``` shell
-$ cat ~/my-projects/servo/.cargo/config
-paths = ['../glutin', '../cocoa-rs']
+If the output is in the format `https://github.com/servo/webrender#0.11.0`, you are dealing with a git dependency and you will have to edit the `~/my-projects/servo/Cargo.toml` file and add at the bottom:
```

Comment:
nope, that's  what `./mach cargo pkgid webrender` prints, although indeed in the `replace` statement one needs to ad a `:`